cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/RunCommand.java

```diff
@@ -182,7 +195,8 @@ public void printLongDesc(StringBuilder out) {
 
     @Override
     public void printUsage(StringBuilder out) {
-        out.append("  bal run [--offline] [<balfile> | <project-path> | executable-jar]\n" +
+        out.append("  bal run <executable-jar>\n");
```

Comment:
Since the `run` command is set to work with `--debug` would it make sense to modify this line to say `bal run [--debug] <executable-jar>` or should I include it in a new line?

Reply:
yes, let's modify the same without adding a new line

Reply:
We should do the same change to the `runCmd` constant at L88